libtsuba/include/tsuba/RDG.h

```diff
@@ -180,15 +177,21 @@ class KATANA_EXPORT RDG {
   /// Explain to graph how it is derived from previous version
   void AddLineage(const std::string& command_line);
 
+  /// Add a new RDGTopology instance to the RDG
+  void AddTopology(tsuba::RDGTopology topo);
+
   /// Load the RDG described by the metadata in handle into memory.
   static katana::Result<RDG> Make(RDGHandle handle, const RDGLoadOptions& opts);
 
-  katana::Result<void> UnbindTopologyFileStorage();
-
   /// Inform this RDG that its topology is in storage at this location
   /// without loading it into memory. \param new_top must exist and be in
   /// the correct directory for this RDG
-  katana::Result<void> SetTopologyFile(const katana::Uri& new_top);
+  /// *This function is only for use in graph conversion, and will deprecated*
```

Comment:
Could you clarify this comment a little? What do you mean by graph conversion, why does that mean it can/will be deprecated?

Reply:
This comment was a little out of date, updated comment and function name to better reflect what it does.